subsys/bluetooth/controller/ll_sw/nordic/lll/lll_central_iso.c

```diff
@@ -577,7 +577,7 @@ static void isr_tx(void *param)
 
 	/* Schedule next subevent */
 	if (se_curr < cis_lll->nse) {
-		const struct lll_conn *conn_lll;
+		const struct lll_conn *conn_lll2;
```

Comment:
@cvinayak suggestions? 

(generally adding a number after a variable name is a terrible choice IMO)

Reply:
I renamed this one to `evt_conn_lll`.